ceno_host/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+edition.workspace = true
+license.workspace = true
+name = "ceno_host"
+version.workspace = true
+
+[dependencies]
+rkyv = { version = "0.8.9", features = ["unaligned"] }
```

Comment:
I need to check whether going without 'unaligned' improves performance in the guest.

It's bit more hassle to make that work, though.